src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/PersonAddress.kt

```diff
@@ -1,18 +1,32 @@
 package uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa
 
+import java.time.LocalDate
 import javax.persistence.DiscriminatorValue
 import javax.persistence.Entity
-import javax.persistence.FetchType
+import javax.persistence.FetchType.LAZY
 import javax.persistence.JoinColumn
 import javax.persistence.ManyToOne
 
 @Entity
 @DiscriminatorValue(PersonAddress.ADDR_TYPE)
-data class PersonAddress(
```

Comment:
why is this no longer a `data` class?

Reply:
@petergphillips I think I will keep it as a plain class; this inherits from a plain class so we just don't have any of the benefits of a data class; there is only one attribute on the old data class - and making it a data class forces you to add an ugly constructor to allow for the parameters that need to be passed to the super class.